src/main/scala/circt/stage/phases/CIRCT.scala

```diff
@@ -148,8 +148,20 @@ class CIRCT extends Phase {
 
     try {
       logger.info(s"""Running CIRCT: '${cmd.mkString(" ")} < $$input'""")
-      val result = (cmd #< new java.io.ByteArrayInputStream(input.getBytes)).!!
-
+      val stdoutStream = new java.io.ByteArrayOutputStream
+      val stderrStream = new java.io.ByteArrayOutputStream
+      val stdoutWriter = new java.io.PrintWriter(stdoutStream)
+      val stderrWriter = new java.io.PrintWriter(stderrStream)
+      val exitValue = (cmd #< new java.io.ByteArrayInputStream(input.getBytes)).!(ProcessLogger(stdoutWriter.println, stderrWriter.println))
+      stdoutWriter.close()
+      stderrWriter.close()
+      val result = stdoutStream.toString
+      val errors = stderrStream.toString
+      if (exitValue != 0) {
+        //logger.error(errors)
+        StageUtils.dramaticError(s"${binary} failed.\nExitCode:\n${exitValue}\nSTDOUT:\n${result}\nSTDERR:\n${errors}")
```

Comment:
@seldridge @jackkoenig not sure the preference for using `logger.error()` vs `StageUtils.dramaticError`